packages/edit-post/src/components/header/style.scss

```diff
@@ -1,19 +1,22 @@
 .edit-post-header {
-	height: $header-height;
 	padding: $grid-size-small 2px;
 	border-bottom: $border-width solid $light-gray-500;
 	background: $white;
 	display: flex;
-	flex-direction: row;
-	align-items: stretch;
+	flex-flow: row wrap;
 	justify-content: space-between;
+	max-width: 100vw;
 	z-index: z-index(".edit-post-header");
 	left: 0;
 	right: 0;
 
-	// Stick the toolbar to the top, because the admin bar is not fixed on mobile.
-	top: 0;
-	position: sticky;
+	// Make toolbar sticky on larger breakpoints
+	@include break-mobile {
```

Comment:
Is there a reason you removed the sticky feature from mobile? In doing so, the publish and preview buttons require you to swipe all the way to the top to do so. Before:

![before-scrolling](https://user-images.githubusercontent.com/1204802/60247115-5218a380-98c0-11e9-8ff2-f2699353aef3.gif)

After:

![after-scrolling](https://user-images.githubusercontent.com/1204802/60247119-547afd80-98c0-11e9-9b4c-2f705eb00ed4.gif)

Changing this behavior is not off the table — a cogent argument can be made that there is insufficient editing space especially when the soft keyboard is showing. However I would think this change needs discussion separately, unless it is absolutely critical in order to make this PR work.

Reply:
The reason I removed the sticky behaviour from mobile is that it was explicitly flagged in issue #15346 as a problem to be solved:

> For the post header set the position attribute to block instead
of sticky when a media query detects the viewport is small (20-30
em).

Design feedback was requested on that issue in early May and @mapk approved the proposed solution, which included the quote above. So I assumed it was safe to implement this solution :-)

For context, the 20-30 em mentioned should give us 280-420px at our body font size of 14. That is a bit smaller than our smallest breakpoint, which is 480px, but I was reluctant to add a new breakpoint. 

It may be justified to do so though, to cater explicitly to large zoom levels, so I propose adding a 280px breakpoint, which is more or less equivalent to a 640px screen width at 250% zoom. That's the point beyond which the top bar elements start to wrap.

Reply:
Thank you for resurfacing that for me, I missed this. 

Given that context, I think that a new "micro" 280px seems like a clever "best of both worlds" solution. If you feel this PR can handle it, that could work, otherwise a separate followup PR can do it.

Reply:
Done!